packages/connect-form/src/components/advanced-options-tabs/authentication-tab/authentication-default.tsx

```diff
@@ -1,9 +1,181 @@
-import React from 'react';
+import React, { useCallback } from 'react';
+import {
+  Icon,
+  IconButton,
+  Label,
+  RadioBox,
+  RadioBoxGroup,
+  TextInput,
+  css,
+  spacing,
+} from '@mongodb-js/compass-components';
+import ConnectionStringUrl from 'mongodb-connection-string-url';
+import { AuthMechanism } from 'mongodb';
+
+import { UpdateConnectionFormField } from '../../../hooks/use-connect-form';
+import FormFieldContainer from '../../form-field-container';
+import { ConnectionFormError } from '../../../utils/validation';
+
+const authSourceLabelStyles = css({
+  padding: 0,
+  margin: 0,
+  flexGrow: 1,
+});
+
+const infoButtonStyles = css({
+  verticalAlign: 'middle',
+  marginTop: -spacing[1],
+});
+
+const defaultAuthMechanismOptions: {
+  title: string;
+  value: AuthMechanism;
+}[] = [
+  {
+    title: 'Default',
+    value: AuthMechanism.MONGODB_DEFAULT,
+  },
+  {
+    title: 'SCRAM-SHA-1',
+    value: AuthMechanism.MONGODB_SCRAM_SHA1,
+  },
+  {
+    title: 'SCRAM-SHA-256',
+    value: AuthMechanism.MONGODB_SCRAM_SHA256,
+  },
+];
+
+function AuthenticationDefault({
+  errors,
+  connectionStringUrl,
+  updateConnectionFormField,
+}: {
+  connectionStringUrl: ConnectionStringUrl;
+  errors: ConnectionFormError[];
+  updateConnectionFormField: UpdateConnectionFormField;
+}): React.ReactElement {
+  const password = connectionStringUrl.password;
```

Comment:
Is this relevant here? https://jira.mongodb.org/browse/COMPASS-5097

can we at least add a test or 2 with the weird password there?

Reply:
Could we do that work when we do that ticket? This work is definitely related, but since we have a ticket for it, that might be a better place to make a pr, test all of the cases and make those changes.